pynio/service.py

```diff
@@ -53,6 +55,26 @@ def stop(self):
         """ Stops the nio Service. """
         self._instance._get('services/{}/stop'.format(self._name))
 
+    def command(self, *args, **kwargs):
+        '''send a command to the service or to the block.
+        To the service:
+            service.command('command')
+        To a block in the service:
+            service.command(block, 'command')
+
+        kwargs are passed onto the request. Some of use are:
+            data: add data onto the request
+            timeout: set the request timeout
+        '''
+        get = self._instance._get
+        cmd_structure = '{}/{}/{}'.format
+        if isinstance(args[0], Block):
+            blk, cmd = args
+            return get(cmd_structure('blocks', blk._name, cmd), **kwargs)
```

Comment:
I don't think the concept of running commands on a non-running block event exists. I think a better interface for the service command method would be command(service, command). It might make more sense to just have a separate method called command_block.